libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -383,10 +383,10 @@ - (BOOL)logoutOnInvalidCredentials:(NSError *)error
 - (NSURL *)fullFileUrlForPage:(NSString *)page
 {
     NSString *fullPath = [[[[NSBundle mainBundle] resourcePath] stringByAppendingPathComponent:self.wwwFolderName] stringByAppendingPathComponent:page];
-    if (![[NSFileManager defaultManager] fileExistsAtPath:fullPath]) {
```

Comment:
It doesn't make a difference in this particular class, but might make a difference in ```UserAccountManager``` or ```SmartStore```:

Per apple documentation using ```[[NSFileManager alloc] init]``` is better than ```[NSFileManager defaultManager]```, since the latter returns the same instance and can lock up.